	_COMPLEX_DEFINED should be defined for MSVC, otherwise
	complex is redefined inconsistently in <math.h>.

	Mail from Akira, 31 Mar 2012 21:36:26.

diff -ur pmx-2.6.17.orig/libf2c/f2c.h pmx-2.6.17/libf2c/f2c.h
--- pmx-2.6.17.orig/libf2c/f2c.h	2012-03-26 12:09:53.000000000 +0200
+++ pmx-2.6.17/libf2c/f2c.h	2012-04-01 14:50:19.000000000 +0200
@@ -13,6 +13,9 @@
 typedef short int shortint;
 typedef float real;
 typedef double doublereal;
+#ifdef WIN32
+#define _COMPLEX_DEFINED 1
+#endif
 typedef struct { real r, i; } complex;
 typedef struct { doublereal r, i; } doublecomplex;
 typedef long int logical;
